From a83165bc06dfa86d0b52c7cb1d93afa69baad0f7 Mon Sep 17 00:00:00 2001
From: Michael Biebl <biebl@debian.org>
Date: Tue, 5 Mar 2013 04:32:30 +0100
Subject: [PATCH 06/10] Setup tmpfile to comply with existing Debian practice

- Clean /tmp on boot but not for files older then 10 days
- Do *not* clean /var/tmp regularly
- Use 1777 root:root since Debian doesn't have a lock group
---
 tmpfiles.d/legacy.conf |    4 ++--
 tmpfiles.d/tmp.conf    |    4 ++--
 2 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/tmpfiles.d/legacy.conf b/tmpfiles.d/legacy.conf
index 3fff347..7f3206c 100644
--- a/tmpfiles.d/legacy.conf
+++ b/tmpfiles.d/legacy.conf
@@ -10,7 +10,7 @@
 # These files are considered legacy and are unnecessary on legacy-free
 # systems.
 
-d /run/lock 0755 root root -
+d /run/lock 1777 root root -
 
 # /run/lock/subsys is used for serializing SysV service execution, and
 # hence without use on SysV-less systems.
@@ -23,7 +23,7 @@ d /run/lock/subsys 0755 root root -
 # On modern systems a BSD file lock is a better choice if
 # serialization is needed on those devices.
 
-d /run/lock/lockdev 0775 root lock -
+d /run/lock/lockdev 0755 root root -
 
 # /forcefsck, /fastboot and /forcequotecheck are deprecated in favor of the
 # kernel command line options 'fsck.mode=force', 'fsck.mode=skip' and
diff --git a/tmpfiles.d/tmp.conf b/tmpfiles.d/tmp.conf
index 3b534a1..74bc677 100644
--- a/tmpfiles.d/tmp.conf
+++ b/tmpfiles.d/tmp.conf
@@ -8,8 +8,8 @@
 # See tmpfiles.d(5) for details
 
 # Clear tmp directories separately, to make them easier to override
-d /tmp 1777 root root 10d
-d /var/tmp 1777 root root 30d
+D /tmp 1777 root root -
+#d /var/tmp 1777 root root 30d
 
 # Exclude namespace mountpoints created with PrivateTmp=yes
 x /tmp/systemd-private-*
-- 
1.7.10.4

